.github/workflows/docs.yml

```
@@ -146,7 +146,7 @@ jobs:
         shell: cmd /D /E:ON /V:OFF /C "CALL "{0}""
         working-directory: .ci
         run: |
-          build_doc.bat > ..\doc\log.txt && type ..\doc\log.txt 2>&1
+          build_doc.bat
```

```suggestion
          build_doc.bat> ..\doc\log.txt && type ..\doc\log.txt 2>&1
```
@luisaFelixSalles c'était à cause de ce changement, je le revert